scripts/schema/exclude_filter.py

```diff
@@ -0,0 +1,62 @@
+import glob
```

Comment:
Do `glob`, `yaml`, or `os` modules get called directly here? If not, we can clean up these imports.

Reply:
nope, removed unused imports here and also `loader.py`and `subset_filter.py`